tests/integration-tests/tests/schedulers/test_slurm_accounting.py

```diff
@@ -123,6 +123,14 @@ def _test_that_slurmdbd_is_running(remote_command_executor):
     assert_that(_is_accounting_enabled(remote_command_executor)).is_true()
 
 
+def _test_slurm_accounting_password(remote_command_executor):
```

Comment:
As you said, this should be addressed on the cookbook since the behavior we are testing is that of a recipe.

However fixing the cookbook now is not strictly necessary, so it may be acceptable to start with this change in an existing test as long as
* we keep track of this need (with a comment here and a Task to move this test in the cookbook)
* we begin to review the integration tests and the cookbook